packages/charts/src/charts/donut/donut-total.component.tsx

```diff
@@ -12,32 +12,34 @@ const DonutTotal: FC<Props> = memo(({ children, ...typography }) => {
   const { fontSize, fontFamily } = rest;
   const totalLabelSize = 14;
```

Comment:
Why it's 14 not for example 22 or 13 ? :)

Reply:
font size is taken from UI https://projects.invisionapp.com/d/#/console/19037699/396877841/preview

Reply:
should it be editable as other labels?

Reply:
IMO - yes as hard coding this value here could be a misleading in future.